README.md

```diff
@@ -11,25 +11,29 @@ This topic explains options, variables, querying, and other options specific to
+| `Name`                              | The data source name. This is how you refer to the data source in panels and queries.                                   |
+| `Default`                           | Default data source means that it will be pre-selected for new panels.                                                  |
+| `Authentication Provider`           | Specify the provider to get credentials.                                                                                |
+| `Access Key ID`                     | If `Access & secret key` is selected, specify the Access Key of the security credentials to use.                        |
+| `Secret Access Key`                 | If `Access & secret key` is selected, specify the Secret Key of the security credentials to use.                        |
+| `Credentials Profile Name`          | Specify the name of the profile to use (if you use `~/.aws/credentials` file), leave blank for default.                 |
+| `Assume Role Arn` (optional)        | Specify the ARN of the role to assume.                                                                                  |
+| `External ID` (optional)            | If you are assuming a role in another account, that has been created with an external ID, specify the external ID here. |
+| `Endpoint` (optional)               | Optionally, specify a custom endpoint for the service.                                                                  |
+| `Default Region`                    | Region in which the cluster is deployed.                                                                                |
+| `AWS Secrets Manager`               | To authenticate with Amazon Redshift using AWS Secrets Manager.                                                         |
+| `Temporary credentials`             | To authenticate with Amazon Redshift using temporary database credentials.                                              |
+| `Serverless`                        | To use a Redshift Serverless workgroup.                                                                                   |
+| `Cluster Identifier`                | Redshift Provisioned Cluster to use (automatically set if using AWS Secrets Manager).                                   |
+| `Workgroup`                         | Redshift Serverless Workgroup to use.                                                                                   |
+| `Managed Secret`                    | When using AWS Secrets Manager, select the secret containing the credentials to access the database. Note that Provisioned and Serverless stores credentials in a different format. Refer to [Storing database credentials in AWS Secrets Manager](https://docs.aws.amazon.com/redshift/latest/mgmt/data-api-access.html#data-api-secrets) for instructions. |
```

Comment:
one last request if you can: can we give examples of what these 2 formats are?

Reply:
Going to merge as is, so we don't block this anymore, can always fix up docs in another pr!